roles/acm_hypershift/defaults/main.yml

```diff
@@ -2,11 +2,11 @@
 ah_cluster_network_cidr: 10.132.0.0/14
 ah_clusters_ns: clusters
 ah_cluster_type: "kubevirt"
+ah_cpo_annotations: ""
```

Comment:
This var is not documented in the README

Reply:
So, may we add this to the README in the "Variables for disconnected environments" section?

Reply:
No, because is internal, the one that is public is `ah_hc_annotations` and that one is documented in that section.

Perhaps it could help to rename that variable without the prefix and that would make it clear that is not an incoming variable, I'll do that.

But what I must do is to move the `ah_hc_annotations` as that one can be used in both scenarios (dis)connected.

I'll get that fixed :+1: